install.php

```diff
@@ -161,7 +161,7 @@ function print_form($d){
     include(DOKU_CONF.'license.php');
 
     if(!is_array($d)) $d = array();
-    $d = array_map('htmlspecialchars',$d);
+    $d = array_map('hsc',$d);
```

Comment:
Did you check if hsc is available in install.php already? It used to be that install.php had not the full environment initialized, yet.